azure-pipelines.yml

```diff
@@ -7,7 +7,13 @@ trigger:
 
 # Still allow PR builds to run against main
 pr:
-- main
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    # Only run the pipelines if a meta.yaml recipe was modified.
+    - recipes/*/meta.yaml
```

Comment:
Are these changes cumulative per PR, or per commit? Wondering case a push only contains changes to e.g. `build.sh`.

Reply:
THey are cummulative. I just committed your change and the workflow still triggered.